src/textual/app.py

```diff
@@ -2448,16 +2448,14 @@ async def _prune_node(self, root: Widget) -> None:
         self._unregister(root)
 
     async def action_check_bindings(self, key: str) -> None:
+        """Handle a key press using the bindings system."""
```

Comment:
Could you look at the other actions in app. I've linked `actions` to the docs. I think we should follow this convention.